openembedded-core-mbl/scripts/signctl/src/signctl.py

```diff
@@ -141,32 +152,35 @@ def parse_args():
     )
     dump_cmd.set_defaults(func=handle_gen_cmd)
 
-    sign_cmd = commands.add_parser("sign-tfa", help="Sign TF-A BL and FIP.")
-    sign_cmd.add_argument(
-        "--rpi-armstub8", type=abspath, help="Path to armstub8.bin"
+    sign_cmd = commands.add_parser(
+        "sign", help="Sign TF-A bootloader components and u-boot FIT binary."
     )
     sign_cmd.add_argument(
-        "--fip",
+        "--rpi-vc4-fw",
         type=abspath,
-        nargs="+",
-        required=True,
-        help="Path to one or more FIPs to be signed.",
+        metavar="PATH",
+        help="Path to a Raspberry-PI 3 VC4 firmware image (e.g armstub8.bin) "
+        "to be signed. Note, TF-A BL2 is also in the VC4 firmware image. "
+        "To sign BL2 on rapsberrypi3 use this option, not the --bl2 option."
```

Comment:
```suggestion
        "To sign BL2 on raspberrypi3 use this option, not the --bl2 option."
```

Reply:
So - should we catch when both bl2 and rpi-vc4-fw are used together?  Or is that in some way valid?

Reply:
Added a mutually exclusive argument group in 139aff9